website/blog/2020-02-31-version-60.md

```diff
@@ -0,0 +1,61 @@
+
+## CocoaPods by Default
+
+[CocoaPods are now part of React Native's iOS project](https://github.com/react-native-community/discussions-and-proposals/blob/master/proposals/0004-cocoapods-support-improvements.md). If you weren't already, be sure to open iOS platform code using the `xcworkspace` file from now on. Also, the `podspec`s for the internal packages have changed to make them compatible with the Xcode projects, which will help with troubleshooting and debugging. Expect to make some straightforward changes to your `Podfile` as part of the upgrade to 0.60 to bring this exciting support.
+
+## Lean Core Removals
+
+**WebView** and **NetInfo** were previously extracted into separate repositories, and in 0.60 we’ve finished migrating them out of the React Native repository. Additionally, in response to community feedback about new App Store policy, **Geolocation** has been extracted. If you haven’t already, complete your migration by adding dependencies to [react-native-webview](https://github.com/react-native-community/react-native-webview), [@react-native-community/netinfo](https://github.com/react-native-community/react-native-netinfo), and [@react-native-community/geolocation](https://github.com/react-native-community/react-native-geolocation). Maintainers have made more than 100 commits to these repositories since extraction and we’re excited to see the community’s support!
+
+## Native Modules are now Autolinked
+
+The team working on the [React Native CLI](https://github.com/react-native-community/cli) has introduced major improvements to native module linking called [autolinking](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md)! Most scenarios will not require the use of `react-native link` anymore. At the same time, the team overhauled the linking process in general. Let us know what you think using the hashtag [#autolinkingInRN](https://twitter.com/intent/tweet?hashtags=autolinkingInRN).
+
+## Upgrade Helper
+
+[@lucasbento](https://github.com/lucasbento), [@pvinis](https://github.com/pvinis), [@kelset](https://github.com/kelset), and [@watadarkstar](https://github.com/watadarkstar) have built a great tool called [Upgrade Helper](https://react-native-community.github.io/upgrade-helper/) to make the upgrade process simpler. It helps React Native users with brownfield apps or complex customizations to see what's changed between versions. Take a look at the [upgrade from 0.59.9 to 0.60.0](https://react-native-community.github.io/upgrade-helper/?from=0.59.9&to=0.60.0) and try it out today for your upgrade path!
```

Comment:
I'm working on #1063, it would be nice to add a link to the section here that mentions how to use it.